### PR TITLE
Non-numeric values validation. Proper http response handling.

### PR DESCRIPTION
**Pull request by mkuculyma**
Fix for #16. Additionally http response parsing was improved in order to handle different types of responses (the response is not always in json format).

### CODE REVIEW

File: opentsdb/opentsdb.go [hunk 1]
```diff
@@ -125,7 +125,7 @@ func (p *opentsdbPublisher) Publish(contentType string, content []byte, config m
 
 		// Omits invalid data points
 		if temp.Valid() {
-			pts[i] = temp
+			pts = append(pts, temp)
 			i++
 		}
```
Comment (IzabellaRaulin) [inline opentsdb/opentsdb.go]: It will be great to have some log here with information that string data has been omitted.

**Comment by andrzej-k** [comment]
Please update godeps, from Travis: "fatal: reference is not a tree: 4f43a8a1f5c43939300148ec0a12e58bc18d1f29" - there is no such commit-id in snap tree.

**Comment by andrzej-k** [comment]
Please update date in license headers: "Copyright 2015 Intel Corporation" -> "Copyright 2015-2016 Intel Corporation"

**Comment by andrzej-k** [comment]
Please update unit tests to cover your changes.

**Comment by IzabellaRaulin** [comment]
I think @andrzej-k meant file-headers like [this](https://github.com/mkuculyma/snap-plugin-publisher-opentsdb/blob/b005d018ab1df21d11afb8a8053ba5969f747927/opentsdb/httpclient.go#L5)

**Comment by andrzej-k** [comment]
LGTM

**Pull request merged**